resources/translations/cs_CZ/troubleshooting/unfinished_print.md

```diff
@@ -8,7 +8,7 @@ Poruchy tiskárny
 ----
 Příčinou neúplného tisku je někdy selhání tiskárny, když tiskárna nedělá to, co se od ní očekává. Obecně platí, že Cura může udělat jen málo pro nápravu těchto problémů. Zde jsou některé běžné příčiny.
 * Výpadek napájení. Cura s tím nemůže dělat nic jiného než rychlejší tisk, aby se snížilo riziko výpadku napájení během tisku.
-* Porucha napájení. Pokud je jedna z teplot (buď [podložky tisku](../material/material_bed_temperature.md) nebo [trysky](../material/material_print_temperature.md)), příliš vysoká, některé tiskárny mají poruchový režim, kde napájecí zdroj shoří nebo teplotní senzory vydávají chyby. Tiskárna se poté vypne a nedokončí tisk.
```

Comment:
This is removing a closing bracket, so the brackets don't match any more. The bracketed text starts with the word `buď`.

Reply:
fixed